translations/en-GB.json

```
@@ -616,6 +616,9 @@
   "NOT_A_PATRON": "You are not currently a patron and have no Starboard Premium to redeem. If you are a patron, make sure you've [connected your Discord account to your Patreon account](https://support.patreon.com/hc/en-us/articles/212052266-Getting-Discord-access).",
   "PATRON_FOR_A_DIFFERENT_TIER": "Although you are a patron, you cannot redeem Starboard Premium as you are not pledged to the Starboard Premium tier and are pledged to a different tier.",
   "ALREADY_REDEEMED_PREMIUM": "It seems you've already redeemed premium in one of your servers.",
+  "ASK_TO_MOVE_PREMIUM": "Do you wish to move your premium subscription to this server?  You will lose premium on {guild}.",
+  "MOVE_PREMIUM_MOVE_HERE_BUTTON": "Move Premium Subscription here",
+  "MOVE_PREMIUM_MOVE_SUCCESS": "Your Premium Subscription has been moved to this server and removed from {guild}.",
```

Gotta limit the number of injected English (bc obviously Starboard is gonna be translated into tons and tons of languages)
Also just added clarification that the action is reversible
```suggestion
  "ASK_TO_MOVE_PREMIUM": "Do you wish to move your premium subscription to this server?  Any items (overrides, reward roles, etc.) that exceed the free limit on {guild, select, null {your server with ID `{guildID}`} other {**{guild}**}} will be saved and will be restored the next time Starboard Premium is redeemed in that server.",
  "MOVE_PREMIUM_MOVE_HERE_BUTTON": "Move Premium Subscription here",
  "MOVE_PREMIUM_MOVE_SUCCESS": "Your Premium Subscription has been moved to this server and removed from {guild, select, null {your server with ID `{guildID}`} other {**{guild}**}}.",
```